buf/registry/legacy/federation/v1beta1/upload_service.proto

```diff
@@ -41,44 +41,40 @@ service UploadService {
 // See the package documentation for more details. You should likely use buf.registry.module.v1beta1
 // and not this package.
 message UploadRequest {
-  // A dependency of Content, either referencing another Content message, or referencing
-  // a Commit that already exists.
+  // A dependency of one or more references specified by Contents.
+  //
+  // Dependencies between Contents are implicit and do not need to be specified. The BSR will detect
+  // dependencies between Contenta via .proto imports.
   message DepRef {
-    // The Module of the dep.
-    buf.registry.module.v1beta1.ModuleRef module_ref = 1 [(buf.validate.field).required = true];
```

Comment:
I think having both `ModuleRef` + `commit_id` helps making more explicit that I depend on a commit of _that_ module.

Reply:
That's duplicate information - a commit_id represents a Commit, and a Commit has a specific module. The rest of our API returns Commits, which have a module ID, as an example. Specifying duplicate information on an API request is an anti-pattern